api/pkg/git/git.go

```diff
@@ -74,7 +74,14 @@ func (c *client) Fetch(spec FetchSpec) (Repo, error) {
 }
 
 func (c *client) initRepo(spec FetchSpec) (*LocalRepo, error) {
-	log := c.log.With("name", "repo").With("url", spec.URL)
+
+	cloneUrl := spec.URL
+
+	if spec.SSHUrl != "" {
```

Comment:
so SSHURL will be given more priority if specified ?

Reply:
as of now yes. Or should we try to attempt with both the urls?